module_utils/network_lsr/argument_validator.py

```diff
@@ -2012,3 +2083,101 @@ def _ipv6_is_not_configured(connection):
                             "match.path is not supported by the running version of "
                             "NetworkManger.",
                         )
+        self.validate_route_tables(connection, idx)
+
+
+class IPRouteUtils(object):
+
+    ROUTE_TABLE_ALIAS_RE = re.compile("^[a-zA-Z0-9_.-]+$")
+
+    @classmethod
+    def _parse_route_tables_mapping(cls, file_content, mapping):
+        for line in file_content.split(b"\n"):
+
+            #
+            # We are a bit more forgiving, but the aliases must be nice UTF-8 names.
+
+            # iproute2 only skips over leading ' ' and '\t'. We are more forgiving,
+            # and strip all leading+trailing whitespace.
+            line = line.strip()
+
+            # skip empty lines or comments
+            if not line:
+                continue
+            if line[0] == ord(b"#"):
+                continue
+
+            # Split at the first space. We accept here any whitespace (unlike iproute2).
+            # Also, this will strip any leading/trailing whitespaces (unlike iproute2)
+            ll = line.split(maxsplit=1)
```

Comment:
I think this is not compatible with python 2.7. The implementation in python 2.7 is not using keywords.

https://docs.python.org/2.7/library/stdtypes.html#str.split

Reply:
I removed the keyword.